i18n/messages/en.json

```
@@ -1294,6 +1274,11 @@
     "description": "Fieldset preview content description",
     "originalDefault": "Fieldset content"
   },
+  "h9i++U": {
+    "defaultMessage": "When this is checked, Open Forms automatically updates the customer profile when a digital address is added or the preferred digital address is changed.",
```

```suggestion
    "defaultMessage": "When this is checked, Open Forms automatically updates the customer profile when a digital address is added.",
```

I think we decided not to update a preferred address and say that the use can update it themselves later